CRM/Utils/SQL/TempTable.php

```diff
@@ -96,6 +100,7 @@ public static function build() {
     // I suspect it would be better to just say utf8=true, but a lot of existing queries don't do the utf8 bit.
     $t->utf8 = CRM_Utils_Constant::value('CIVICRM_TEMP_FORCE_UTF8', FALSE);
```

Comment:
I would like to address this in a future PR, as I think we could always make UTF-8 the default character set, and rely on column-specific statements if we need something else like an ASCII column, case-sensitive collation, etc.

Reply:
@mdb I agree - I suggested we set it to TRUE when @totten added it - he left it as false out of caution but I'd be happy to change it - say just after the next rc drops so it gets a long cycle in master/rc

Reply:
Well I was thinking we could remove the utf8 property altogether? I can't think of a need to specify a different default. Aside from having a different default for the whole database in general.

Reply:
I guess so but changing the default feels like a safer step

Reply:
Well, the fact that we could change the default kindof proves that we don't really need the toggle at all :)

Reply:
@mfb OK - so I guess to prove that point we should
1) test setting the define ourselves (pref on prod)
2) ensure buildkit is setting it so all demo builds are getting it 
3) maybe the same for new installs

Reply:
I made this PR to experiment with setting the constant to TRUE and maybe remove it entirely: #13658    I don't really care too much what the eventual solution is, but it seemed like something that could be simplified :)